health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppLocalisationTable.js

```
@@ -21,40 +22,42 @@ const Tabs = ({ availableLocales, onTabChange, setActiveLocale, activeLocale })
-          />
-        );
-      },
-    },
+    ...((Array.isArray(availableLocales) ? availableLocales : []).map((locale) => ({
+      name: t(locale?.label || locale?.value || ""),
+      cell: (row) => (
+        <TextInput
+          name={`translation_${locale?.value || ""}`}
+          value={(row && locale?.value && row[locale.value]) || ""}
+          onChange={(event) => {
+            if (row && row.code && locale && locale.value && typeof updateLocalization === "function") {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)